qlib/data/dataset/handler.py

```diff
@@ -222,7 +222,7 @@ def _fetch_data(
         # This method is extracted for sharing in subclasses
         from .storage import BaseHandlerStorage  # pylint: disable=C0415
 
-        # Following conflictions may occurs
+        # Following conflations may occur
```

Comment:
What does it mean by conflation?
The documents want to describe a confliction case.